test/integration-test/build.rs

```diff
@@ -116,10 +116,15 @@ fn main() {
             target_arch.push(arch);
         };
 
+        let libbpf_vmlinux_dir = libbpf_dir.join(".github/actions/build-selftests");
```

Comment:
This could use a comment describing what is going on here. It's clear this is the path to a vmlinux.h, but what is not clear is:

1. How the file was generated?
2. Is this location stable? It looks like it's part of a GH action in libbpf, but from some simple grepping it doesn't appear to be used anywhere.

Is this something we want to do on all platforms?

Reply:
Done. Yes, we want to do it on all platforms.

Reply:
> // However in our tests we do not make use of kernel data structures, and so any vmlinux.h
> // which defines the constants we need (e.g. `__u8`, `__u64`, `BPF_MAP_TYPE_ARRAY`,
> // `BPF_ANY`, `XDP_PASS`, `XDP_DROP`, etc.) will suffice.

If it is just a handful of contants, why not just write a quick `linux.h`?
That would keep things self-contained.

Reply:
Yeah, we could. This is a shortcut, and we can take the long-cut if and when we need to. Does that seem reasonable?